actions/templates/post/index.js

```diff
@@ -20,6 +20,28 @@ const consoleLib = require('@adobe/aio-lib-console');
 const HTTP_METHOD = 'post';
 const POST_PARAM_NAME = 'name';
 
+/**
+ * Serialize the request body
+ * @param {object} params action params
+ * @returns {object} serialized request body
+ */
+const serializeRequestBody = (params) => {
+  return {
+    name: params.name,
+    ...(params.description && { description: params.description }), // developer console only
+    ...(params.version && { version: params.version }), // developer console only
+    ...(params.createdBy && { createdBy: params.createdBy }),
+    ...(params.author && { author: params.author }), // developer console only
+    ...(params.status && { status: params.status }), // developer console only
+    ...(params.adobeRecommended && { adobeRecommended: params.adobeRecommended }), // developer console only
+    ...(params.codeSamples && { codeSamples: params.codeSamples }), // developer console only
+    links: {
+      ...(params?.links?.consoleProject && { consoleProject: params.links.consoleProject }), // developer console only
```

Comment:
I saw in some test data in another PR, `links` can contain an `npm` property also?

Reply:
Ah yes `links` can have `npm`, however it's not set via request body, but inferred from the template name when inserting a new template [here](https://github.com/adobe/adp-template-registry-api/blob/main/actions/templateRegistry.js#L55)